server/graphql/types/src/types/program/encounter.rs

```diff
@@ -101,25 +101,29 @@ pub struct EncounterFilterInput {
     pub clinician_id: Option<EqualFilterStringInput>,
     pub document_name: Option<EqualFilterStringInput>,
     pub document_data: Option<StringFilterInput>,
+    pub patient: Option<PatientFilterInput>,
+    pub program: Option<ProgramEnrolmentFilterInput>,
 }
 
-impl EncounterFilterInput {
-    pub fn to_domain_filter(self) -> EncounterFilter {
+impl From<EncounterFilterInput> for EncounterFilter {
+    fn from(f: EncounterFilterInput) -> Self {
```

Comment:
I'm not sure. I wasn't part of this conversation I don't think? 

So I could use into e.g. ProgramEnrolmentFilter::into....

Reply:
Fine to leave it. Using `From` has the problem that its a bit hard to track down the implementation while with a special method you can just go to the method definition. Since these `Froms` are only used once they don't provide any extra advantages for which reason a to_domain method is easier...